README.md

```diff
@@ -22,6 +22,17 @@ of the following ways:
 - Create a [Github issue](https://github.com/pulibrary/dls-handbook/issues)
 - Message us in #digital_library in PULibrary's Slack
 
+## Our Vision
+
+Princeton University Library has defined its [Mission, Vision, and North Star
+Statements](https://library.princeton.edu/about) and DLS has worked with its
+partners to define the Digital Library's part in the wider vision and how we can
+best move it forward. It is as follows:
```

Comment:
This sentence is pretty long. I took a stab at breaking it into 2 simpler sentences.

DLS has worked with its partners to define a vision for the Digital Library in the context of Princeton University Library's Mission Vision, and North Star Statements. This Digital Library vision helps us situate our work within the broader vision, and helps us move that vision forward. It is as follows:

Reply:
Remove this from start of sentence, perhaps? and capitalize vision (in line with how the vision is capitalized for PUL)

Our Digital Library Vision  helps us.... or
The Digital  Library Vision helps us...

Reply:
The draft felt a little duplicative, how does...

```
DLS has worked with its partners to define a vision for the Digital Library in
the context of Princeton University Library's [Mission, Vision, and North Star
Statements](https://library.princeton.edu/about) to help us situate
our work within the broader vision and move it forward. It is as
follows:
```

sound?